Source/Manifest/manifest-xml.env

```diff
@@ -0,0 +1,261 @@
+_t4dManifestXmlGetLine(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates.env
+# v1.8.0
+###### _t4dManifestXmlGetLine
+#   - Name:
+#       _t4dManifestXmlGetLine
+###
+#   - Synopsis:
+#       Get line in manifest xml file or in input
+###
+#   - Example:
+#       * {Example1}              = {One-Line Description}
+#
+###
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
+###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
+    local _t4dErrorCode=1
+
+    local _type=$1
+    local _nameType=$2
+    local _name=$3
+    local _filename="$4"
+
+    if [[ -e "${_filename}" ]]; then
+        _filename="$(cat $_filename)"
+    fi
+
+    echo "$_filename" | grep "<${_type} " | grep " $_nameType=\"${_name}\"" 
+
+###############################
+}
+
+_t4dManifestXmlGetData(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates.env
+# v1.8.0
+###### _t4dManifestXmlGetData
+#   - Name:
+#       _t4dManifestXmlGetData
+###
+#   - Synopsis:
+#       Get data in manifest xml file or in input
+###
+#   - Example:
+#       * _t4dManifestXmlGetData <type> <typeName> <Name> <Data> [ManifestPath]              = Find data in xml file
+#
+###
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
+###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
+    local _t4dErrorCode=1
+
+    local _type=$1
+    local _nameType=$2
+    local _name=$3
+    local _attribute=$4
+    local _filename="$5"
+
+    if [[ -e "${_filename}" ]]; then
```

Comment:
you copy paste those content, instead, please consider creating a specific function that will return xml content and check that sended content/file is a manifest